apps/01PigLatin.js

```diff
@@ -4,13 +4,67 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // Your Answer:
+  //.toLowerCase makes each letter of a string a lowercase letter
+
+  word = word.toLowerCase();
+
+  // Concept Check
+  // What is the significance of setting vowelIndex to -1?
+  // Your Answer:
+  //setting vowelIndex to -1 assumes that there are no vowels in the string before checking them with the if statements
+  //this allows us to step through the if statements if there is no value of each vowel in the string
+
```

Comment:
Close. vowelIndex === -1 means we haven't found a vowel, and we start with that as our initial condition. If we end up finding a vowel, vowelIndex will be updated with a value other than -1.